File: typescript/easy/kids-with-the-greatest-number-of-candies.ts
function kidsWithCandies(candies: number[], extraCandies: number): boolean[] {
    const maxCandies = Math.max(...candies);
    const ans: boolean[] = [];

    for (let i = 0; i < candies.length; i++) {
        ans.push(candies[i] + extraCandies >= maxCandies);
    }

    return ans;
}

File: csharp/easy/kids-with-the-greatest-number-of-candies.cs
IList<bool> KidsWithCandies(int[] candies, int extraCandies)
{
    int maxCandies = candies.Max();
    List<bool> ans = new List<bool>();

    for (int i = 0; i < candies.Length; i++)
    {
        ans.Add(candies[i] + extraCandies >= maxCandies);
    }

    return ans;
}
